app/javascript/components/Articles/ArticlesHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+
+type ArticleHeaderProps = {
+    title:string,
+    subtitle:string
+}
+
+class ArticleHeader extends React.Component<ArticleHeaderProps> {
```

Comment:
Another class that could be a functional component.

Reply:
Interesting to learn about the different types of components and when may be best to use them.